modules/ROOT/pages/mule-4-api-javadoc.adoc

```diff
@@ -1,9 +1,20 @@
-= Mule API Javadoc
+= Mule Runtime Engine Javadoc
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: java, classes, javadoc, reference, objects, methods
 
-== Java Classes
+MuleSoft provides Javadocs for the Community Edition of the Mule Runtime Engine.
+The Javadoc contains definitions for classes, fields and their data types, and also describes methods including their usage, parameters and return values.
 
-* https://www.mulesoft.org/docs/site/4.1.1/apidocs/[Mule 4.1.1 API]
\ No newline at end of file
+Read the documentation for Mule SDK if you want an easier way to extend the Mule Runtime Engine to suit your needs.
```

Comment:
pls do an inline link to the Mule SDK docs

Reply:
suggestion: "the documentation for" is not needed

suggestion: "suit your needs" might be difficult for some readers (esp some people who speak English as a second language) to understand. Try something more common, like "meet your needs"

Reply:
I've added the link to Mule SDK to avoid using inline links, in this case do you think it is better to have it inline?

Reply:
cool, looks good